wrappers/javascript/aries-askar-nodejs/tests/migration.test.ts

```diff
@@ -0,0 +1,51 @@
+import { ariesAskar, Migration } from '@hyperledger/aries-askar-shared'
+import fs from 'fs'
+import path from 'path'
+
+const DB_TEMPLATE_PATH = path.join(__dirname, 'indy_wallet_sqlite.db')
+const DB_UPGRADE_PATH = path.join(__dirname, 'indy_wallet_sqlite_upgraded.db')
+
+describe('migration', () => {
+  beforeAll(() => {
+    require('@hyperledger/aries-askar-nodejs')
+  })
+
+  beforeEach(() => {
+    const tplPaths = [DB_TEMPLATE_PATH, `${DB_TEMPLATE_PATH}-shm`, `${DB_TEMPLATE_PATH}-wal`]
```

Comment:
what is `tpl` and `upd` ? `templatePaths` and `updatePaths`?

Reply:
Yep, same comment as above, copied from `tests/migration.rs`.